src/core/admin-core.js

```diff
@@ -8,14 +8,38 @@ function deleteFeedItem(id) {
   .then(deletedCount => {
     if (deletedCount > 1) {
       logger.error('Deleted feed item', id);
-      throw new Error('Unexpected amount of deletes happened: ' + deletedCount)
+      throw new Error('Unexpected amount of deletes happened: ' + deletedCount);
     }
 
     return deletedCount;
   });
 }
 
+function shadowBanUser(id) {
```

Comment:
Pitäisikö tätä nimeämistä jotenkin yhdenmukaistaa? Shadowban voisi olla ihan vain Ban?
Ei siis koske niinkään tätä PRää vaan yleisesti tuli tätä lukiessa mieleen.

Reply:
Ban riittäis varmaan ihan hyvin